VideoWeb/VideoWeb/ClientApp/src/app/services/user-media.service.ts

```diff
@@ -199,7 +200,21 @@ export class UserMediaService {
             take(1),
             map(stream => !!stream && stream.getVideoTracks().length > 0 && stream.getAudioTracks().length > 0),
             catchError(error => {
+                console.log(error);
```

Comment:
this extra console log probably isnt needed as you as you're logging the error already on the line below